BDArmory/FX/ExplosionFX.cs

```diff
@@ -998,7 +998,7 @@ private void ExecutePartBlastEvent(PartBlastHitEvent eventToExecute)
                                         {
                                             WarheadTypes.ShapedCharge => 5000f,
                                             WarheadTypes.Kinetic => ImpactSpeed,
-                                            _ => ExplosionVelocity
+                                            _ => ExplosionVelocity //technically this should be the sum vector of the explosion vel (perpendicular to missile vel), and missile vel since the rods are physical projectiles that would be inheriting their parent's vel
```

Comment:
It would have to be a separate value if doing it that way, since the are a few places in ExplosionFx that need the static value. Standard warheads wouldn't need it, since they don't make use of ``ExplosionVelocity``, and CR would only need it for 982 and 1001. At any rate, it'd be something for a later PR.

Reply:
Fair enough. We can leave it as just a comment for now then.